Header header

geometry_msgs/Pose world_T_jk
geometry_msgs/Pose world_T_cf
geometry_msgs/Twist jk_twist

geometry_msgs/Pose world_T_jk_target
geometry_msgs/Pose world_T_cf_target
geometry_msgs/Pose world_T_cf_future


